### PR TITLE
[DX-862] Add additional information about enterprise portal version for cloud

### PR DESCRIPTION
**Pull request by vverbani**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/frequently-asked-questions/long-term-support-releases.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ Occasionally, we may see a need to issue a critical fix if there is a systems do
 
 The only exception to this policy is if we ever need to release a breaking change. This would mean that we have to release a new major version (i.e. releasing version 2.0). In this exceptional circumstance we would support both the old major version and the new one concurrently for six months - please note that the old version only gets supported in terms of critical fixes, not new functionality. After the six months is up, the previous major version falls out of support.
 
+If you are using the enterprise portal on Tyk Cloud, we recommend you use the latest version at all times. When a new release is launched, older versions are immediately deprecated. This gives you the best, latest, and most secure capability. If a customer wants to use a previous version, this is possible via rollback; but if an issue arises on an older version, we wouldn't patch that version; we'd advise you to upgrade to latest version
```
Comment (dcs3spp) [inline tyk-docs/content/frequently-asked-questions/long-term-support-releases.md]:
```suggestion
If you are using the Enterprise Portal on Tyk Cloud, we recommend you use the latest version at all times. When a new release is launched, older versions are immediately deprecated. This gives you the best, latest and most secure capability. If a customer wants to use a previous version, this is possible via rollback. However, if an issue arises on an older version, we wouldn't patch that version; we'd advise you to upgrade to latest version
```

**Review by dcs3spp — APPROVED** [review]

File: tyk-docs/content/frequently-asked-questions/long-term-support-releases.md [hunk 2]
```diff
@@ -77,6 +77,8 @@ Occasionally, we may see a need to issue a critical fix if there is a systems do
 
 The only exception to this policy is if we ever need to release a breaking change. This would mean that we have to release a new major version (i.e. releasing version 2.0). In this exceptional circumstance we would support both the old major version and the new one concurrently for six months - please note that the old version only gets supported in terms of critical fixes, not new functionality. After the six months is up, the previous major version falls out of support.
 
+If you are using the Enterprise Portal on Tyk Cloud, we recommend you use the latest version at all times. When a new release is launched, older versions are immediately deprecated. This gives you the best, latest and most secure capability. If a customer wants to use a previous version, this is possible via rollback. However, if an issue arises on an older version, we wouldn't patch that version; we'd advise you to upgrade to latest version
```
Comment (excieve) [inline tyk-docs/content/frequently-asked-questions/long-term-support-releases.md]:
```suggestion
If you are using the Enterprise Portal on Tyk Cloud, we recommend you use the latest version at all times. When a new release is launched, older versions are immediately deprecated. This gives you the best, latest and most secure capability. If a customer wants to use a previous version, we won't enforce an upgrade unless there are operational or security reasons to do so. However, if an issue arises on an older version, we wouldn't patch that version; we'd advise you to upgrade to latest version.
```

**Review by excieve — CHANGES_REQUESTED** [review]
Thanks @vverbani! As discussed internally, let's change the rollback line a bit to better reflect reality. Otherwise looks good!

**Review by excieve — APPROVED** [review]

**Pull request merged**